tfjs-backend-webgl/src/texture_manager.ts

```diff
@@ -25,13 +25,13 @@ export class TextureManager {
   private numUsedTextures = 0;
   private numFreeTextures = 0;
   private _numBytesAllocated = 0;
-  private _numBytesFree = 0;  // How many bytes that have been allocated
-                              // are available for reuse.
-  private freeTextures: {[shape: string]: Texture[]} = {};
+  // Number of bytes that have been allocated and available for reuse.
+  private _numBytesFree = 0;
+  private freeTextures?: Record<string, Texture[]> = {};
+  private usedTextures?: Record<string, Texture[]> = {};
```

Comment:
Nit: Why is this optional (with `?`) but immediately assigned? Does it get deleted somewhere else?

Reply:
Yes. `TextureManager.dispose()` assigns nulls to them, and a lot of tests relies on this behavior (set to null after disposed) to be correct, at least they failed when I tried to make it always not null. There may be some additional guards/checks needed after we turn on the null check, but so far it's fine since it matches the old behavior.

Reply:
Sounds good. Thanks!

Reply:
Rolled back and added a todo comment